### ISSUE TITLE
Missing tags/typo

### ISSUE THREAD

**Issue by tommygebru** [jboutiere/prj-rev-bwfs-dasmoto#5]
Your paragraph tag is not closed in your html div with classname “brushes"

**Comment by jboutiere** [comment]
Good catch. Is there a way this typo would manifest in the display? I did use the visual display of my page as my check of the code. Does the lack of the closing tag not have an effect on the display of just this particular code?

**Comment by tommygebru** [comment]
Hey sorry I didnt see this notification, so there are times, when an error does not visibily manifest itself and thats really cool. I believe that the browser itself will at times help and prevent some breaking as well. But as always stick to the best practices and inspecting your code well 👍
